### PR TITLE
Adding checkmk to One-Click-Apps

### PR DESCRIPTION
**Pull request by drewOP**
Added checkmk raw edition to One-Click-Apps
https://checkmk.com/

### ☑️ Self Check before Merge

- ☑️ I have tested the template using the method described in README.md thoroughly
- ☑️ I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- ☑️ I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- ☑️ I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- ☑️ Icon is added as a png file to the logos directory.
- ☑️ I've executed the checks if necessary by running `npm ci && npm run validate_apps && npm run formatter`

### CODE REVIEW

File: public/v4/apps/checkmk.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+captainVersion: 4
+services:
+    '$$cap_appname':
+        image: checkmk/check-mk-raw:$$cap_version
+        environment:
+            TZ: '$$cap_tz'
+            PHP_TZ: '$$cap_tz'
```
Comment (githubsaturn) [inline public/v4/apps/checkmk.yml:7]: I don't see these two env vars in checkmk's docs. Also the associated CapRover variable doesn't exist in the variables section.

File: public/v4/apps/checkmk.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        volumes:
+            - '$$cap_appname-data:/omd/sites'
+            - /etc/localtime:/etc/localtime:ro
```
Comment (githubsaturn) [inline public/v4/apps/checkmk.yml:10]: `:ro` is not read by CapRover. Can be removed.

File: public/v4/apps/checkmk.yml (hunk 1 above)
Comment (drewOP) [reply public/v4/apps/checkmk.yml]: just some copy and paste errors. Fixed it.

File: public/v4/apps/checkmk.yml (hunk 2 above)
Comment (drewOP) [reply public/v4/apps/checkmk.yml]: removed it

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**